src/main/scala/za/co/absa/hyperdrive/trigger/configuration/application/KafkaConfig.scala

```diff
@@ -0,0 +1,39 @@
+
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.hyperdrive.trigger.configuration.application
+
+import org.springframework.boot.context.properties.bind.Name
+import org.springframework.boot.context.properties.{ConfigurationProperties, ConstructorBinding}
+import org.springframework.validation.annotation.Validated
+
+import java.util.Properties
+import javax.validation.constraints.{NotBlank, NotNull}
+import scala.annotation.meta.field
+
+@ConfigurationProperties("kafka-source")
```

Comment:
Should not be prefix kafkaSource ?

Reply:
No, this is another convention. https://docs.spring.io/spring-boot/docs/current/reference/html/features.html#features.external-config.typesafe-configuration-properties.relaxed-binding
"The prefix value for the annotation must be in kebab case (lowercase and separated by -, such as my.main-project.person)."

Reply:
Thanks